### ISSUE TITLE
Getting error when try to add docker swarm manager into multipass VM

### ISSUE THREAD

**Issue by Ivanshamir** [canonical/multipass#2287]
Hello sir, after launching multipass VM in my machine, I cannot add docker swarm with the assigned IP. In my machine the assigned IP is 10.173.198.201. So when I run this command:
`multipass exec node1 -- /bin/bash -c 'docker swarm init --advertise-addr 10.173.198.201:2377 --listen-addr 10.173.198.201:2377'`
 Then the error comes up. It says:
`Error response from daemon: advertise address must be a non-zero IP address or network interface  (with optional port number)`

Any idea how can I launch multipass without non-zero IP or any custom IP?Is it possible?

**Comment by Saviq** [comment]
Hi @Ivanshamir not sure what the problem is there. Is `10.173.198.201` the IP your instance got? Or where's it coming from?

**Comment by Ivanshamir** [comment]
> Hi @Ivanshamir not sure what the problem is there. Is `10.173.198.201` the IP your instance got? Or where's it coming from?

Absolutely. This is the assigned IP of VM. And the launching command is `multipass launch -n first`
